ocropus-rpred

```diff
@@ -133,6 +133,8 @@ def process1(arg):
     (trial,fname) = arg
     base,_ = ocrolib.allsplitext(fname)
     line = ocrolib.read_image_gray(fname)
+    if not ocrolib.isbinarized(line):
+        raise ValueError("Image %s is not binarized" % (fname))
```

Comment:
I think here we should either replace the current (not working) check above or delete it.